Backend/dev/migrations/rider-app/1275-indexing-on-payout-order-table.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX idx_entity_name ON atlas_app.payout_order USING btree (enitity_name);
```

Comment:
typo in `enitity_name` and what could be possible values for `enitity_name`, looks like not many values will be there for this column, creating index on this would be very costly.

Reply:
sure , will remove this